src/pages/en/core-concepts/astro-components.md

```diff
@@ -175,6 +163,34 @@ const visible = true;
 {visible ? <p>Show me!<p> : <p>Else show me!</p>}
 ```
 
+### Dynamic Tags
+
+Local variables can also represent dynamic HTML _tags_:
+
+```astro title="src/components/DynamicTags.astro" /Tag|<Tag>|</Tag>/
+---
+import MySpecialHeader from "../MySpecialHeader.astro"
+const tags = ["h1", "h2", "h3"];
+---
+<div>
+    Three sizes of headers:
+    {tags.map((Tag) => <Tag>A header!</Tag>)}
+</div>
+```
+
+This also works with components:
+```astro title="src/components/DynamicTags.astro" /BigRedHeader|Tag|<Tag>|</Tag>/
+---
+import BigRedHeader from "./BigRedHeader.astro";
+const tags = ["h1", "h2", "h3", BigRedHeader];
+---
+
+<div>
+    Three sizes of headers:
```

Comment:
```suggestion
    Four different headers:
```
?? I'm honestly not sure this example works, because I have no idea what size the BigRedHeader is and whether it's the same size as one of the existing ones. But, this is going to give you four headers, is it not? And without knowing what the BigRedHeader actuually looks like, maybe they're not all in fact different?

The example kind of feels forced, and I'd almost prefer to see something entirely different that doesn't include any tags, but maybe just maps over an array of related components to make the point.